test/framework/controller/constant.go

```diff
@@ -14,7 +14,14 @@
 package controller
 
 const (
+	// CNI Metrics Helper Constants
 	CNIMetricsHelperChartDir    = "/test/helm/charts/cni-metrics-helper"
 	CNIMetricsHelperReleaseName = "cni-metrics-helper"
 	CNIMetricHelperNamespace    = "kube-system"
+
+	// Calico Constants
+	CalicoNamespace    = "kube-system"
+	CalicoReleaseName  = "aws-calico"
+	CalicoChartName    = "aws-calico"
+	CalicoChartRepoURL = "https://aws.github.io/eks-charts"
```

Comment:
I think we should rely on the charts folder in aws-vpc-cni repo - https://github.com/aws/amazon-vpc-cni-k8s/tree/master/charts. Since now we make the changes in the source repo and then merge it to charts repo.